bin/commands/start.js

```diff
@@ -132,7 +135,7 @@ module.exports = function () {
       }
     })
     .catch(err => {
-      console.error(err);
+      console.log(err.stack);
```

Comment:
Since we're exiting the program with an error code, shouldn't the error stack be printed on stderr?